modules/javafx.swt/src/main/java/javafx/embed/swt/FXCanvas.java

```diff
@@ -246,6 +257,16 @@ static Transfer getCustomTransfer(String mime) {
             } catch (Exception e) {
                 //Fail silently.  If we can't get the methods, then the current version of SWT has no retina support
             }
+        } else if (SWT.getPlatform().equals("win32")) {
+            try {
+                String autoScale = AccessController.doPrivileged((PrivilegedAction<String>)() -> System.getProperty("swt.autoScale"));
+                if(autoScale == null || ! "false".equalsIgnoreCase(autoScale)) {
+                    Class dpiUtilClass = Class.forName("org.eclipse.swt.internal.DPIUtil");
+                    swtDPIUtilMethod = dpiUtilClass.getMethod("getDeviceZoom");
+                }
+            } catch (Exception e) {
+                //Fail silently.  If we can't get the methods, then the current version of SWT has no retina support
+            }            
```

Comment:
I missed this the first time....there is a trailing whitespace on this line that will cause jcheck to fail.